packages/amplify_core/ios/Classes/Support/ErrorUtil.swift

```diff
@@ -33,4 +33,12 @@ public class ErrorUtil {
         serializedException["underlyingException"] = underlyingError
         return serializedException
     }
+    
+    public static func postRestartErrorToFlutterChannel(result: FlutterResult,
+                                          errorCode: HotReloadExceptionCode,
+                                          details: Dictionary<String, String>)  {
+        result(FlutterError(code: errorCode.rawValue,
+                            message: ErrorMessages.defaultFallbackErrorMessage,
+                            details: details))
+    }
```

Comment:
same, not needed? And bunch of other changes from previous revision

Reply:
Fixed.